payload.go

```diff
@@ -137,7 +144,15 @@ func ProcessPayload(e []github.Event, u User) ([]ActivityPayload, error) {
 			if comment, ok := rawPayload["comment"].(map[string]interface{}); ok {
 				payload.Body = comment["body"].(string)
 			}
-			activityPayloads = append(activityPayloads, payload)
+
+			// Save the paylod to the db events table
+			err := payload.Create()
+			if err != nil {
+				//Assuming fail due to the unique constraint
+			} else {
+				activityPayloads = append(activityPayloads, payload)
+			}
+
```

Comment:
I guess I don't follow.  This whole block

			// Save the paylod to the db events table
			err := payload.Create()
			if err != nil {
				//Assuming fail due to the unique constraint
			} else {
				activityPayloads = append(activityPayloads, payload)
			}
 
is repeated for each `case`, so why can't it just happen once at the end.  Nothing needs to be popped off if we dont append it to begin with, right?